[2020-09-26 10:06:26,467] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: core_concepts.python_function 2020-09-25T00:00:00+00:00 [queued]>
[2020-09-26 10:06:26,477] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: core_concepts.python_function 2020-09-25T00:00:00+00:00 [queued]>
[2020-09-26 10:06:26,477] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-09-26 10:06:26,477] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-09-26 10:06:26,478] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-09-26 10:06:26,484] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): python_function> on 2020-09-25T00:00:00+00:00
[2020-09-26 10:06:26,486] {standard_task_runner.py:53} INFO - Started process 97677 to run task
[2020-09-26 10:06:26,543] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: core_concepts.python_function 2020-09-25T00:00:00+00:00 [running]> 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2020-09-26 10:06:26,562] {logging_mixin.py:112} INFO - 0.13436424411240122
[2020-09-26 10:06:26,563] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-09-26 10:06:26,567] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=core_concepts, task_id=python_function, execution_date=20200925T000000, start_date=20200926T170626, end_date=20200926T170626
[2020-09-26 10:06:36,457] {logging_mixin.py:112} INFO - [2020-09-26 10:06:36,456] {local_task_job.py:103} INFO - Task exited with return code 0
